static-checks/nist-validation/test.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
+
+import re
+import requests
+from pathlib import Path
+
+from lib import util, results
+
+
+# Get scapval zip file link from NIST web
+scapval_web = 'https://csrc.nist.gov/Projects/Security-Content-Automation-Protocol/SCAP-Releases/SCAP-1-3' # noqa
+r = requests.get(scapval_web)
+scapval_web_path = re.search(r'href="(.*scapval-.*.zip)"', r.text)
+if not scapval_web_path:
+    raise RuntimeError("SCAPval tool link not found on NIST web")
+scapval_link = 'https://csrc.nist.gov' + scapval_web_path.group(1)
+scapval_zip = scapval_link.split('/')[-1]
```

Comment:
Could we please just create a `contest-data` repository here under `RHSecurityCompliance` and store the zip there?

I seem to remember that being the idea when we were discussing it >1 year ago, as there have been a non-trivial amount of failures due to NIST's site being temporarily unreachable at times. Maybe it was due to some throttling of accesses, maybe outages, but I reckon Github has a much higher data hosting capability.

Reply:
Sure, I can do that. However, are we going to check if our repo contains latest scapval? NIST updates scapval from time to time (very rarely). In our previous scapval test, each run we have been comparing our scapval zip SHA with SHA provided by NIST on their web. It's unnecessary complexity I wanted to avoid, thus chosen the path of downloading it directly from web.

Reply:
Hmm, good point.

On one hand, I would hate either
* having an undocumented process of maybe sometime in the future updating the zip in `contest-data`
* having a documented process of checking for an update, as an extra added complexity for Contest

but I think the real choice is between

1. leaving a potentially old version of scapval in `contest-data`, and updating it when it starts failing (?)
   * without checking any SHA from NIST (would create dependency on NIST being online)
   * this has the advantage of it being under our control, it won't change between stabilization runs during one release, etc.
   * also more reliable hosting
1. always downloading it from NIST

and I think I would still prefer (1), but it's definitely not black-and-white.

I also really don't like the sub-path mangling just to get the zip URL.